jdaviz/configs/cubeviz/plugins/slice/tests/test_slice.py

```diff
@@ -62,8 +72,30 @@ def test_slice(cubeviz_helper, spectrum1d_cube):
     cubeviz_helper.select_wavelength(4.62360028e-07)
     assert sl.slice == 1
 
+    # Test player buttons API
+
+    sl.vue_goto_last()
+    assert sl.slice == sl.max_value
+
+    sl.vue_play_next()  # Should automatically wrap to beginning
+    assert sl.slice == 0
+
+    sl.vue_play_start_stop()  # Start
+    assert sl.is_playing
+    assert sl._player.is_alive()
+    time.sleep(1)  # Give it time to play something
```

Comment:
Might have to bump this or find a way to test the play button that doesn't require sleeping...
```suggestion
    time.sleep(1.5)  # Give it time to play something
```

I'll ponder about this while I wait for replies on where the buttons should really go.